StoreTFstateAzure.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I don't think this file is used anywhere currently - if you plan to leave it for future reference it would be worth at least adding a brief comment to the top of the file with what it's for